run/helm/snap/README.md

```diff
@@ -0,0 +1,246 @@
+
+As you may read in [Snap documentation](https://github.com/intelsdi-x/snap#overview) the framework consists of two main components: `snapteld` daemon and plugins. This chart deploys and evokes only Snap daemon `snapteld`. In order to begin collection of any metrics we need some plugins. It is up to the user which plugin they wish to use. The list of all plugins is available in [Plugin Catalog](https://github.com/intelsdi-x/snap/blob/master/docs/PLUGIN_CATALOG.md). The user needs to download and load plugins. More information on how to configure and load Snap plugins can be found in [Loading Snap Plugins](#7-loading-snap-plugins) and [Deep Dive](#8-deep-dive) sections.
+
+## 3. Prerequisites
+
+* Kubernetes 1.4+ with Beta APIs enabled
+
+## 4. Installing the Chart
+
+To install the chart with the release name my-snap use command:
+
+`$ helm install --name my-snap stable/snap`
+
+This command deploys Snap on the Kubernetes cluster in the default configuration. To learn how to override the default configuration see section [Configuration](#6-configuration).
+
+## 5. Uninstalling the Chart
+
+To uninstall/delete the my-snap daemonset use command:
+
+`$ helm delete my-snap`
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## 6. Configuration
+
+To see the default configuration of Snap chart run:
+
+`$ helm inspect values stable/snap`
+
+To override any of these settings create your own YAML formatted file and pass it during installation:
+```
+$ echo 'host_plugins: /opt/snap' > new_values.yaml
+$ helm  install -f new_values.yaml stable/snap
+```
+or use `set` flag to specify custom values on the command line like this:
+
+`$ helm install --set host_plugins=/opt/snap stable/snap`
+
+You will find more information in Helm Guide section [Customizing the Chart Before Installing](https://github.com/kubernetes/helm/blob/master/docs/using_helm.md#customizing-the-chart-before-installing).
+
+| Parameter                 | Description                                         | Default                           |
+|---------------------------|-----------------------------------------------------|-----------------------------------|
+| `image.repo`              | docker image                                        | intelsdi/snap4kube                |
+| `image.tag`               | docker image tag                                    | latest                            |
+| `image.pullPolicy`        | image pull policy                                   | IfNotPresent                      |
+| `mounts`                  | list of volumes to mount inside the Snap container  | /proc:/proc_host                  |
+| `host_autoload`           | host path to the directory with plugins and tasks   | /opt/autoload                     |
+| `privileged`              | security context needed to access files on the host | false                             |
```

Comment:
Most of Snap collectors need access to "/proc" path when running inside the container. I reviewed Snap plugins and I think that Docker collector is the only one that would require privileged mode (mountpoint of cgroups) when running inside the container (but it would be good thing to confirm that, and test all plugins inside the Docker container). 

So enabling `privileged` mode by default is not recommended, unless we want Docker collector plugin as integral part of Snap chart (it is related to @bodepd bigger question: are we delivering snap or a monitoring stack)